src/main/java/io/teak/sdk/Log.java

```diff
@@ -197,7 +199,12 @@ public Log(String androidLogTag, int jsonIndentation) {
             synchronized (commonPayload) {
                 // Add sdk version to common payload, and log init message
                 commonPayload.put("sdk_version", Teak.Version);
-                log(Level.Info, "sdk_init", null);
+
+                // Log ISO8601 format timestamp at init
+                SimpleDateFormat iso8601 = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US);
+                Map<String, Object> dateTime = new HashMap<>();
+                dateTime.put("at", iso8601.format(new Date()));
```

Comment:
Just confirming, this is definitely UTC?

Reply:
It is for sure now.